package.json

```diff
@@ -138,6 +139,7 @@
     "@angular/compiler-cli": "14.2.0",
     "@angular/language-service": "14.2.0",
     "@angular/localize": "14.2.0",
+    "@angular/pwa": "^14.2.9",
```

Comment:
Can you fix the version to `14.2.9` please and run `yarn install` again: `"@angular/pwa": "14.2.9"`